src/dotnet/commands/dotnet-new/CSharp_xunittest/project.json.template

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24008",
```

Comment:
But I think we just moved to 127.  :smile:.  Off by a couple number shouldn't matter.  The next update-dependencies run will clean them all up to be consistent.